valet-key/Readme.md

```diff
@@ -4,47 +4,61 @@ This document describes the Valet Key Pattern example from the guide [Cloud Desi
-* Open the file app.config from the ValetKey.Client project and change the setting for serviceEndpointUrl to   [your-URL]**/api/values/**
-	* By default this is set to **http://127.0.0.1:81/api/values/**
+	1 - StorageConnectionString
+	2 - StorageKey
 
+	You should be able to get these values from the Azure Portal; go to the storage account, and from the
+	left menu, in the Security + networking section, select the option "Access Keys".
+	In the "Access Keys" dialog, click on "show keys"; you will be able to see and copy the key and the connection string.
 
+* Hit F5 to run the Web Api
 * Start a new instance of the ValetKey.Client project to upload the blob. Right-click the project, select Debug, and click Start new instance.
 
-* If you are running the application locally in the Windows Azure emulator:
-	* Open the Windows Azure Compute Emulator UI from the icon in the notification area.
-	* Select each role in turn and view the diagnostic information generated by Trace statements in the code.
-
-* If you are running the application on Windows Azure:
-	* Open the Server Explorer Window in Visual Studio and expand the Windows Azure entry.
-	* Expand Cloud Services and then expand the entry for the solution you deployed.
-	* Right-click each role instance and select View Diagnostics Data to see the diagnostic information generated by Trace statements in the code. This is written to the WADLogsTable in the Storage/Development/Tables section.
-
-
-
+* If you want to run the example on Windows Azure, provision a Windows Azure App Service and deploy the application to it from Visual Studio.
+* Right click on the ValetKey.Web project, select publish.
+* Select Azure as target and Azure App Service as specific target.
+* Select an app service instance or create a new one.
+* Skip the Api Management step.
+* Once the publish profile is created, in the Hosting section click on the "..." button in the upper right corner.
+* Select "Manage Azure App Service Settings"
+* Add these three settings:
+
+	1 - ContainerName (set the remote value to 'valetkeysample')
+	2 - StorageConnectionString
+	3 - StorageKey
+
+	You should be able to get thes Azure Storage connection string and key values from the Azure Portal; go to the storage account, and from the
+	left menu, in the Security + networking section, select the option "Access Keys".
+	In the "Access Keys" dialog, click on "show keys"; you will be able to see and copy the key and the connection string.
+
+* Run the App Service instance and note the base URL of the web api shown in the browser address bar.
+* Open the file appsettings.json from the ValetKey.Client project and change the setting for ServiceEndpointUrl to   [your-URL]**/api/sas/**
+	* By default this is set to **http://localhost:10194/api/sas** which set up to run locally.
+
+* You can verify that the blob has been sucessfully uploaded by using the Azure Portal.
```

Comment:
```suggestion
* You can verify that the blob has been successfully uploaded by using the Azure portal or Azure Storage Explorer.
```

Reply:
Done, we may be need to go ever the entire readme, though